_common/KeymanHosts.php

```diff
@@ -126,20 +139,23 @@ function __construct() {
         die("tier is '$this->tier' which is invalid\n");
       }
 
+      // Currently, we don't locally host downloads.keyman.com, so
+      // we always have the same address
+      $this->SERVER_downloads_keyman_com = $this->downloads_keyman_com;
+
+      if(empty($this->SERVER_s_keyman_com)) {
+        $this->SERVER_s_keyman_com = $this->s_keyman_com;
+        $this->SERVER_api_keyman_com = $this->api_keyman_com;
+        $this->SERVER_help_keyman_com = $this->help_keyman_com;
+        $this->SERVER_keyman_com = $this->keyman_com;
+        $this->SERVER_keymanweb_com = $this->keymanweb_com;
+      }
       $this->fixupHosts();
     }
 
     private function fixupHosts() {
```

Comment:
If these correspond to ll 34-35, could we rename to fixupSiteHostname()?

Just trying to follow all the uses of "hosts" in this file...

Reply:
I'm hoping to eliminate fixupHosts altogether in a future PR